parser/mpFuncStr.cpp

```diff
@@ -58,7 +58,7 @@ MUP_NAMESPACE_START
     const string_type & str1 = a_pArg[0]->GetString();
     const string_type & str2 = a_pArg[1]->GetString();
 
-    *ret = str1.find(str2) != string_type::npos ? true : false;
+    *ret = str1.find(str2) != string_type::npos;

```

Comment:
Thank you for joining my team now 😆 

## Before

<img width="818" alt="Screen Shot 2023-03-29 at 15 13 36" src="https://user-images.githubusercontent.com/7637806/228630680-8780b078-8ae5-4569-8217-ef4df3d2018b.png">

## After

<img width="546" alt="Screen Shot 2023-03-29 at 15 13 41" src="https://user-images.githubusercontent.com/7637806/228630841-c7165378-abb3-46a7-91fa-ad3f9aac12bf.png">